services/throttle_definitions.proto

```diff
@@ -82,4 +82,6 @@ message ThrottleBucket {
  */
 message ThrottleDefinitions {
   repeated ThrottleBucket throttleBuckets = 1;
+  uint64 totalAllowedGasPerSecFrontend = 2;
```

Comment:
It's also worth noting that we have an (informal) contract with mirror node operators to **never** remove proto fields, no matter how long they have been deprecated. 😲

So if we have even a 10% doubt that maybe we'll end up following @shemnon's suggestion [here](https://github.com/hashgraph/hedera-services/pull/2373#discussion_r741285719), and using a "normal" throttle bucket for gas throttling, then maybe for release 0.20 we just make these two numbers global/dynamic properties.

But if we're pretty certain about splitting these numbers out in file `0.0.123`, then seems fine.

Reply:
Moved the gas limit configs to global dynamic properties.